pkgs/development/r-modules/wrapper-rstudio-server.nix

```
@@ -0,0 +1,40 @@
+{ lib, runCommand, R, fontconfig, rstudio-server, wrapQtAppsHook, recommendedPackages, packages, qtbase }:
+
+let
+  qtVersion = with lib.versions; "${major qtbase.version}.${minor qtbase.version}";
+in
+runCommand (rstudio-server.name + "-wrapper") {
+  preferLocalBuild = true;
+  allowSubstitutes = false;
+
+  nativeBuildInputs = [wrapQtAppsHook];
```

prefered style
```suggestion
  nativeBuildInputs = [ wrapQtAppsHook ];
```
please apply elsewhere